src/blocks/InsuranceInfoBlock/InsuranceInfoBlock.tsx

```diff
@@ -0,0 +1,177 @@
+  ${({ hardBottom }) =>
+    hardBottom &&
+    css`
+      padding-bottom: 0;
+    `};
+  ${({ halfHardBottom }) =>
+    halfHardBottom &&
+    css`
+      padding-bottom: 1.5rem;
+    `};
+`
+
+const InsuranceValueDescription = styled.div`
+  color: ${colorsV3.gray700};
+`
+const InsuranceValue = styled.div`
+  font-size: 1.5rem;
+`
+
+const Link = styled.a`
+  display: block;
+  line-height: 1.75;
+  color: inherit;
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    color: inherit;
+    text-decoration: underline;
+  }
+`
+
+const Cta = styled.a`
+  display: inline-block;
+  border: 1px solid ${colorsV3.gray900};
+  border-radius: 8px;
+  padding: 0.75rem 1rem;
+  width: 100%;
+  color: inherit;
+  text-decoration: none;
+  text-align: center;
+`
+
+interface InsuranceInfoBlockProps extends BrandPivotBaseBlockProps {
+  value_1_description: string
+  value_1_value: string
+  value_2_description: string
+  value_2_value: string
+  value_3_description: string
+  value_3_value: string
+  value_4_description: string
+  value_4_value: string
+  terms_link: string
+  terms_link_text: string
+  presale_info_link: string
+  presale_info_link_text: string
+  cta_text: string
+  cta_link: string
+}
+
+export const InsuranceInfoBlock: React.FC<InsuranceInfoBlockProps> = ({
+  color,
+  index,
+  extra_styling,
+  ...values
+}) => {
+  return (
+    <StyledSectionWrapper
+      extraStyling={extra_styling}
+      colorComponent={color}
+      brandPivot
+    >
+      <StyledContentWrapper index={index}>
+        <Column>
+          <InsuranceValueDescription>
+            {values.value_1_description}
+          </InsuranceValueDescription>
+          <InsuranceValue>{values.value_1_value}</InsuranceValue>
+        </Column>
+        <Column>
+          <InsuranceValueDescription>
+            {values.value_2_description}
+          </InsuranceValueDescription>
+          <InsuranceValue>{values.value_2_value}</InsuranceValue>
+        </Column>
+        <Column>
+          <InsuranceValueDescription>
+            {values.value_3_description}
+          </InsuranceValueDescription>
+          <InsuranceValue>{values.value_3_value}</InsuranceValue>
+        </Column>
+        <Column>
+          <InsuranceValueDescription>
+            {values.value_4_description}
+          </InsuranceValueDescription>
+          <InsuranceValue>{values.value_4_value}</InsuranceValue>
+        </Column>
+
+        <Column lastOnMobile hardBottom>
+          <Link
+            href={values.terms_link}
+            target="_blank"
+            rel="noopener nofollow"
+          >
+            {values.terms_link_text}↗
+          </Link>
+          <Link
+            href={values.presale_info_link}
+            target="_blank"
+            rel="noopener nofollow"
+          >
+            {values.presale_info_link_text}↗
+          </Link>
+        </Column>
+
+        <Column halfHardBottom>
+          <Cta href={values.cta_link}>{values.cta_text}</Cta>
```

Comment:
I think we can use the Button component here :) 
```suggestion
          <ButtonLinkBrandPivot href={values.cta_link} fullWidth styleType="outlined">
            {values.cta_text}
          </ButtonLinkBrandPivot>
```

Reply:
oh sorry missed this comment 😅